src/octopeer-github/test/DatabaseAdapterTest.ts

```diff
@@ -0,0 +1,65 @@
+///<reference path="../main/DatabaseAdapter.ts"/>
+
+describe("A DatabaseAdapter", function() {
+    let adapter: DatabaseAdapter;
+
+    beforeEach(function() {
+        jasmine.Ajax.install();
+
+        adapter = new DatabaseAdapter("http://localhost:8000");
+        jasmine.Ajax.requests.mostRecent().respondWith({
+            contentType: "text/json",
+            responseText: '{"url":"http://localhost:8000/api/users/42/","username":"Travis"}',
+            status: 201,
+        });
+    });
+
+    afterEach(function() {
+        delete adapter;
+        jasmine.Ajax.uninstall();
+    });
+
+    it("correctly saves the url with trailing slash", function() {
+        expect(adapter.url).toBe("http://localhost:8000/");
+    });
+
```

Comment:
Maybe also add a test that no additional trailing slash is added when one is already present.